lib/q-transformer.js

```diff
@@ -52,6 +53,10 @@ function createQTransformer(
                 transformerOptions.options = instruction.options;
             }
         }
+        transformerOptions.options = merge(
```

Comment:
If we're moving everything in to the template, is this required for anything? If would be good to have no coupling between the UI Schema in CW and the UI Schema in the template

Reply:
Merge replaced with if statement as discussed (with page schemas taking priority)